data/templates/ssh/sshd_config

```diff
@@ -64,21 +64,33 @@ PrintLastLog yes
 ClientAliveInterval 60
 AcceptEnv LANG LC_*
 
+# Disallow user without ssh or sftp permissions
+AllowGroups ssh.main sftp.main admins root
```

Comment:
Now, remember why I implemented a mechanism to enable this feature (and I disabled it by default). The issue with this is that a user manually created by the admin (with `useradd`) will be blocked. And I think by example I in ynh-dev with virtualbox there are some year ago, a user was created by ynh-dev and by this PR this user will be blocked. In the ynh-dev case it's not a really big issue, but we can see some other user in production which have a same setup like this.

Reply:
Hmmm imho the fact that a user created with useradd doesn't automagically get ssh access is a good thing ? :thinking:

Reply:
For me it's OK, and it improve really the security, as for me actually it like a security holes. But we just need to know that there might be some impacted user by this change.

Reply:
Yup indeed :+1: